cmd/client.go

```diff
@@ -50,42 +67,43 @@ type CloudproberClient struct {
 //	  - nil: No error occurred and the client was successfully initialised.
 //	  - gRPC Error: See https://godoc.org/google.golang.org/grpc/codes for error codes.
 //		-> See https://github.com/grpc/grpc-go/blob/d25c71b54334380ff1febd25d88064b36de44b3c/clientconn.go#L123
-func (cpc *CloudproberClient) InitClient(rpcServer string) error {
-	cpc.clientMux.Lock()
-	defer cpc.clientMux.Unlock()
+func (client *CloudproberClient) initClient(rpcServer string) error {
+	client.clientMux.Lock()
+	defer client.clientMux.Unlock()
+	var err error
```

Comment:
Generally we don't declare `err`, just use `:=` the first time you need it.

Reply:
I have to declare `err` here as it's used in line 76:

`client.conn, err = grpc.Dial(...)`

I can't use `:=` here because `client.conn` is already defined.

Reply:
I've refactored this function and integrated it into `NewClient()` and removed the need for `var err error`.

Reply:
Do you still need to push that change?

Reply:
Can we remove the `var err error` here, and instead declare it with `:=` on 78?

Also, if there are no other callers of `CloudProber.initClient()`, could we just inline it to `NewClient()`?

Reply:
Sorry, my last commit didn't push. I'll push now. I've removed `initClient()`